docs/reference/utility-apis/AlertApi.md

```diff
@@ -67,13 +67,13 @@ export type Observable&lt;T&gt; = {
 </pre>
 
 Defined at
-[packages/core-api/src/types.ts:53](https://github.com/spotify/backstage/blob/0406ace29aba7332a98ff9ef9feedd65adc75223/packages/core-api/src/types.ts#L53).
+[packages/core-api/src/types.ts:53](https://github.com/spotify/backstage/blob/1f010f7828f47d37a523bb06ba8e2c16f5ef7d3a/packages/core-api/src/types.ts#L53).
 
 Referenced by: [alert\$](#alert).
 
 ### Observer
 
-This file contains non-react related core types used through Backstage.
+This file contains non-react related core types used throught Backstage.
```

Comment:
oof :) `throughout` i guess it should have been :)

Reply:
👍 , actually convenient to fix in the PR just to avoid regenerating all docs again